src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -52,13 +52,12 @@
    * order to be relevance and max results to be 100, the API's arbitrary limit
    */
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
     try {
-      YouTube youtubeService = getService();
       String url = request.getParameter(URL_PARAMETER);
       CommentThreadListResponse commentResponse = generateYouTubeRequest(url).execute();
       // TODO: input commentResponse into commentAnalysis class to get useful data
-      Statistics statistics = new Statistics(new ArrayList<Double>(), 0);
+      Statistics statistics = new Statistics(new ArrayList<Double>());
```

Comment:
missing import statistics

Reply:
I think it doesn't have to import statistics, since statistics is in a sub-folder of where this is stored. However, on my current branch, statistics has not been moved to a ".utils" subfolder, so I don't know if this will cause an issue when merged.